### PR TITLE
Import SRP page

### LINKED ISSUE #13442: Import SRP Page
- [ ] "learn more" should be capitalized

- [ ]  Update "Learn More" link to a source that explains what the SRP is for,  rather than one about how to recover it from disk in case you lost it and your password.* 

- [ ]  Change the default behavior of the text area to blur the SRP the user inputs 

*requires design input (cc @rachelcope)

### PR DESCRIPTION
**Pull request by VSaric**
fixes: #13442

**Explanation:**  Import SRP page.

**Screencast:**

https://user-images.githubusercontent.com/92527393/153207400-fe490052-bc8f-45da-a796-56d89dc4efdb.mov

### CODE REVIEW

**Comment by mirjanaKukic** [comment]
Verified by QA

**Review by adonesky1 — DISMISSED** [review]
LGTM!

File: ui/pages/onboarding-flow/import-srp/import-srp.js [hunk 1]
```diff
@@ -56,11 +56,11 @@ export default function ImportSRP({ submitSecretRecoveryPhrase }) {
             <a
               key="learnMore"
               type="link"
-              href="https://metamask.zendesk.com/hc/en-us/articles/360036464651"
+              href="https://metamask.zendesk.com/hc/en-us/articles/360060826432-What-is-a-Secret-Recovery-Phrase-and-how-to-keep-your-crypto-wallet-secure"
```
Comment (brad-decker) [inline ui/pages/onboarding-flow/import-srp/import-srp.js:59]: [REQ] Sorry for this, i know its a nit but it is pretty helpful...could we move this to the zendesk urls constant in the constants folder?

**Review by brad-decker — DISMISSED** [review]

File: ui/pages/onboarding-flow/import-srp/import-srp.js (hunk 1 above)
Comment (VSaric) [reply ui/pages/onboarding-flow/import-srp/import-srp.js]: @brad-decker Fixed in: aad65a42b3056b558751a14d92e6edc5c7aceeba

**Review by jpuri — APPROVED** [review]
LGTM

**Review by adonesky1 — APPROVED** [review]
LGTM

**Pull request merged**